python_modules/dagster/dagster/_core/execution/context/system.py

```diff
@@ -864,6 +864,10 @@ def op_config(self) -> Any:
         op_config = self.resolved_run_config.ops.get(str(self.node_handle))
         return op_config.config if op_config else None
 
+    @property
+    def is_graph_asset_op(self) -> bool:
```

Comment:
in combination with `is_sda_step` this will tell us if an op is in a graph backed asset.